sdk/servicebus/service-bus/src/util/utils.ts

```diff
@@ -180,3 +181,108 @@ export function getAssociatedReceiverName(
   }
   return receiverName;
 }
+
+/**
+ *  @ignore
+ * Helper utility to retrieve `string` value from given input,
+ * or undefined if not passed in.
+ * @param value
+ */
+export function getStringOrUndefined(value: any): string | undefined {
+  if (value) {
+    return value.toString();
+  } else {
+    return undefined;
+  }
+}
+
+/**
+ *  @ignore
+ * Helper utility to retrieve `number` value from given string or undefined,
+ * or undefined if not passed in.
+ * @param value
+ */
+export function getNumberOrUndefined(value: any): number | undefined {
+  if (value) {
+    try {
+      return parseInt(value.toString());
```

Comment:
What if the number is a float?

Reply:
The usage in the feature doesn't deal with float, updated name to use `getInteger..` instead